### ISSUE TITLE
When opening Electron application on Mac - “the git command requires command line developer tools. Would you like to install the tools now?". How to prevent users from seeing this message?

### ISSUE THREAD

**Issue by vikramnayak85** [electron-userland/electron-builder#3026]
**Version**: 20.15.1 
**Target**: mac - dmg
**macOS versions used for application testing**: 10.13.5, 10.11.6, 10.12.6
 
**Issue**: When opening Electron application on Mac - “**the git command requires command line developer tools. Would you like to install the tools now?**”. **I need to get rid of this message**. 

**Description**: 
I have built my Electron application on Mac using Electron builder. The dmg file installs correctly and runs correctly, but each time i open the application, i get the same prompt:

"the git command requires command line developer tools. Would you like to install the tools now?"

I have tried on multiple macs - they all show the same message when i open the application. I have looked everywhere for references to a git command - i only found the "hosted-git-info" package. Any idea what could be causing this issue ?

**Comment by lilliesAndRoses** [comment]
Could you please comment on the steps that you took to solve this issue?

**Comment by vikramnayak85** [comment]
The source of the issue was a package I used  called "systeminformation", which provided details about the OS, device, etc. One of the things that it was doing in the "osinfo.js" file within the module  was running a git command to get the git version (if installed). If git wasn't installed, it gave the prompt to install Git. That's where the issue arose. So I modified the osinfo.js file by commenting out the code for the git command
